travis/test_server

```diff
@@ -239,13 +239,18 @@ def main():
             # Find errors, except from failed mails
             errors = has_test_errors(
                 "stdout.log", database, odoo_version, check_loaded)
+            if returncode != 0:
+                all_errors.append(to_test)
+                print(fail_msg, "Command exited with code %s" % returncode)
+                # If not exists errors then
+                # add an error when returcode!=0
+                # because is really a error.
+                if not errors:
+                    errors += 1
             if errors:
```

Comment:
@dreispt 

Hello, imagine next case:
```python
errors = 0
returncode = -1
```
with
```python
if errors or returncode != 0:
    counted_errors += errors
    all_errors.append(to_test)
    print(fail_msg, "Found %d lines with errors" % errors)
```

You will see next message:
```python
"Found 0 lines with errors"
```

With current patch:
```python
    if returncode != 0:
        all_errors.append(to_test)
        print(fail_msg, "Command exited with code %s" % returncode)
        # If not exists errors then
        # add an error when returcode!=0
        # because is really a error.
        if not errors:
            errors += 1
    if errors:
        counted_errors += errors
        all_errors.append(to_test)
        print(fail_msg, "Found %d lines with errors" % errors)
```
You will see next message:
```python
"Found 1 lines with errors"
```
And variable `counted_errors` It could be used in `expected_errors`
I had the following error in the [first change](https://github.com/vauxoo-dev/maintainer-quality-tools/commit/5282391fa12484432b98db765f043ee98de9b253)
https://travis-ci.org/OCA/maintainer-quality-tools/jobs/59302545